.github/workflows/build.yml

```diff
@@ -77,116 +77,58 @@ jobs:
         name: reactos-${{matrix.compiler}}-${{matrix.arch}}-${{github.sha}}
         path: build/livecd.iso
 
-  build-clang-cl-i386:
-    name: Clang-CL (i386)
+  build-msvc:
+    strategy:
+      matrix:
+        compiler: [msvc, clang-cl]
+        arch: [i386, amd64]
+      fail-fast: false
     runs-on: windows-latest
     steps:
-    - name: Install packages
+    - name: Install ninja
+      run: choco install -y ninja
+    - name: Install LLVM (x86)
```

Comment:
Prefer
```suggestion
    - name: Install LLVM (i386)
```

Reply:
Well, chocolatey says "--x86", let's respect that.